stix_shifter/stix_transmission/src/modules/guardium/guardiumapiclient.py

```diff
@@ -0,0 +1,448 @@
+                    '{"access_token":"' + str(access_token) + '", "expiresTimestamp":' + str(expiresTimestamp) + "'}")
+#
+        host_port = connection.get('host') + ':' + \
+            str(connection.get('port', ''))
+
+        url_modifier_function = None
+        proxy = connection.get('proxy')
+        if proxy is not None:
+            proxy_url = proxy.get('url')
+            proxy_auth = proxy.get('auth')
+            if (proxy_url is not None and proxy_auth is not None):
+                self.headers['proxy'] = proxy_url
+                self.headers['proxy-authorization'] = 'Basic ' + proxy_auth
+            if proxy.get('x_forward_proxy', None) is not None:
+                self.headers['x-forward-url'] = 'https://' + \
+                    host_port + '/'  # + endpoint, is set by 'add_endpoint_to_url_header'
+                host_port = proxy.get('x_forward_proxy')
+                if proxy.get('x_forward_proxy_auth', None) is not None:
+                    self.headers['x-forward-auth'] = proxy.get(
+                        'x_forward_proxy_auth')
+                self.headers['user-agent'] = 'UDS'
+                url_modifier_function = self.add_endpoint_to_url_header
+
+        self.client = RestApiClient(host_port,
+                                    None,
+                                    connection.get('cert', None),
+                                    self.headers,
+                                    url_modifier_function,
+                                    connection.get('cert_verify', 'True')
+                                    )
+
+    def add_endpoint_to_url_header(self, url, endpoint, headers):
+        # this function is called from 'call_api' with proxy forwarding,
+        # it concatenates the endpoint to the header containing the url.
+        headers['x-forward-url'] += endpoint
+        # url is returned since it points to the proxy for initial call
+        return url
+
+    def add_header_elements(self, key, value):
+        self.headers[key] = value
+        return
+
+    def ping_box(self):
+        # Subroto -- Guardium does not have ping facility
+        # We test if we can get the access token if we can then success = true
+        #
+        logging.info("--------- Guardium ping_box ------\n")
+        respObj = Response()
+        if (self.fetch_accessToken()):
+            respObj.code = "200"
+            respObj.error_type = ""
+            respObj.status_code = 200
+            content = '{"status":"OK", "data": {"message": "Service is up."}}'
+            respObj._content = bytes(content, 'utf-8')
+        else:
+            respObj.code = "503"
+            respObj.error_type = "Service Unavailable"
+            respObj.status_code = 503
+            content = '{"status":"Failed", "data": {"message": "Service is down."}}'
+            respObj._content = bytes(content, 'utf-8')
+        # changed by Subroto: https://<server_ip>/api/ariel/searches/<search_id>
```

Comment:
There's a lot of these "changed by Subroto" comments throughout your PR that should be removed. Also, this file has a lot of commented code reference Ariel APIs.

Reply:
I shall remove ariel comments